content/technotes/2018-06-12-ssh-02.md

```diff
@@ -0,0 +1,192 @@
+---
+slug: "ssh-02"
+title: "The ssh Package: Secure Shell (SSH) Client for R"
+date: 2018-06-12
+authors:
+  - name: Jeroen Ooms
+categories: technotes
+cover: "https://upload.wikimedia.org/wikipedia/commons/d/d7/8-cell.gif"
+tags:
+- R
+- packages
+- ssh
+---
+
```

Comment:
I think the introduction could be a bit more engaging if rephrased with questions. "Have you ever needed to connect to a remote server over SSH to transfer files via SCP or to setup a secure tunnel, and wished you could do so from R itself? The new rOpenSci ssh package provides a native ssh client in R allows you to do that and even more, like running a command or script on the host while streaming stdout and stderr directly to the client. Technically, `ssh` is based on libssh, a powerful C library implementing the SSH protocol. "